pubsublite/main_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
```

Comment:
This seems to be a duplicate of internal/wire/main_test. Is there a reason why it's being added here too?

Reply:
Ah yeah, this was unfortunate. `TestMain` only applies to the package where it resides. I will have to copy to a third package, so might move the bulk of it to a common util (probably in a separate PR, since this one is already approved) shared by all the main_test.go clones.